docs/content/guides/operator/indexer-stack-setup.mdx

```diff
@@ -0,0 +1,314 @@
+<td>`obj_versions`</td>
+<td>10-14 days</td>
+<td>unpruned</td>
+</tr>
+<tr>
+<td colspan={5}>
+    <details>
+    <summary>
+    `obj_versions.toml`
+    </summary>
+    <ImportContent source="examples/prod-config/obj_versions.toml" mode="code" />
+    </details>
+</td>
+</tr>
+<tr>
+<td>`tx_affected_addresses.toml`</td>
+<td>Midweight transaction table.</td>
+<td>`tx_affected_addresses`</td>
+<td>1-2 days</td>
+<td>90 days</td>
+</tr>
+<tr>
+<td colspan={5}>
+    <details>
+    <summary>
+    `tx_affected_addresses.toml`
+    </summary>
+    <ImportContent source="examples/prod-config/tx_affected_addresses.toml" mode="code" />
+    </details>
+</td>
+</tr>
+<tr>
+<td>`tx_affected_objects.toml`</td>
+<td>Midweight transaction table.</td>
+<td>`tx_affected_objects`</td>
+<td>1-2 days</td>
+<td>90 days</td>
+</tr>
+<tr>
+<td colspan={5}>
+    <details>
+    <summary>
+    `tx_affected_objects.toml`
+    </summary>
+    <ImportContent source="examples/prod-config/tx_affected_objects.toml" mode="code" />
+    </details>
+</td>
+</tr>
+<tr>
+<td>`tx_calls.toml`</td>
+<td>Midweight transaction table.</td>
+<td>`tx_calls`</td>
+<td>1-2 days</td>
+<td>90 days</td>
+</tr>
+<tr>
+<td colspan={5}>
+    <details>
+    <summary>
+    `tx_calls.toml`
+    </summary>
+    <ImportContent source="examples/prod-config/tx_calls.toml" mode="code" />
+    </details>
+</td>
+</tr>
+<tr>
+<td>`tx_kinds.toml`</td>
+<td>Midweight transaction table.</td>
+<td>`tx_kinds`</td>
+<td>1-2 days</td>
+<td>90 days</td>
+</tr>
+<tr>
+<td colspan={5}>
+    <details>
+    <summary>
+    `tx_kinds.toml`
+    </summary>
+    <ImportContent source="examples/prod-config/tx_kinds.toml" mode="code" />
+    </details>
+</td>
+</tr>
+</tbody>
+</table>
+
+### Hardware requirements
+
+- **CPU:** 8 cores
+- **Memory:** 16GB
+
+## Consistent store setup
+
+All consistent store pipelines run in the same instance based on a single configuration file. Like the indexer, the pipelines run in parallel and throughput is limited by the slowest pipeline.
+
+### Restore command
+
+Restores one or more pipelines from checkpoint data in a GCS bucket.
+
+```sh
+$ sui-indexer-alt-consistent-store restore \
+    --database-path DATABASE_PATH \
+    --http HTTP_ENDPOINT \
+    --object-file-concurrency OBJECT_FILE_CONCURRENCY \
+    --pipeline PIPELINE_NAME
+    --remote-store-url REMOTE_STORE_URL \
+```
+
+| CLI parameter             | Description                                                                                                         |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------|
+| `DATABASE_PATH`           | Path to RocksDB database.                                                                                           |
+| `HTTP_ENDPOINT`           | Path to formal snapshot API.                                                                                        |
```

Comment:
The `--http` option is for public good only. Let's call that out here, because it will be noticeably slower than fetching from the reader-pays bucket in S3.